LongWarOfTheChosen/Config/XComMissionDefs.ini

```diff
@@ -3114,3 +3115,142 @@
 	SquadSpawnSizeOverride=4, \\
 	ForcedTacticalTags[0]="NoVolunteerArmy", \\
 	ForcedTacticalTags[1]="NoDoubleAgent") \\
+
+-arrMissions=(MissionName="ChosenShowdown_Warlock", sType="ChosenShowdown_Warlock", \\
+      MissionFamily="ChosenShowdown_Warlock", \\
+	        MapNames[0]="Obj_ChosenShowdown", \\
+			MapNames[1]="UMS_MissionCore", \\
+			MapNames[2]="UMS_NoMissionTimer", \\
+			MapNames[3]="UMS_NoReinforcements", \\
+			OverrideDefaultMissionIntro=true, \\
+			MissionIntroOverride=( \\
+				MatineePackage="CIN_XP_WarlockShowdownIntro", \\
+				MatineeSequences[0]=(MatineeCommentPrefixes[0]="WarlockShowdown_Intro"), \\
+				MatineeBaseTag="ShowdownIntroMatineeBase"      ), \\
+				RequiredPlotObjectiveTags[0]="ChosenShowdown", \\
+				RequiredParcelObjectiveTags[0]="WarlockShowdown", \\
+				MissionObjectives[0]=(ObjectiveName="EliminateChosen", \\
+				SuccessLootTables[0]=(LootTableName="ChosenWarlockShowdownArtifacts"), \\
+				bIsTacticalObjective=true, bIsStrategyObjective=true, bIsTriadObjective=true), \\
+				MissionObjectives[1]=(ObjectiveName="DestroySarcophagus",      bIsTacticalObjective=true, bIsStrategyObjective=true, bIsTriadObjective=true \\
+				), \\
+			MissionSchedules[0]="ChosenShowdown_Warlock", \\
+			ForcedTacticalTags[0]="NoVolunteerArmy", \\
+			ForcedTacticalTags[1]="NoDoubleAgent",      )
+
++arrMissions=(MissionName="ChosenShowdown_Warlock", sType="ChosenShowdown_Warlock", \\
+      MissionFamily="ChosenShowdown_Warlock", \\
+	        MapNames[0]="Obj_ChosenShowdown", \\
+			MapNames[1]="UMS_MissionCore", \\
+			MapNames[2]="UMS_NoMissionTimer", \\
+			MapNames[3]="UMS_NoReinforcements", \\
+			MapNames[4]="UMS_LWCommon", \\
```

Comment:
Is this the only change? Does it have an remote event listener that will trigger just by being attached to the mission?

Reply:
Yes, it the same way chosen disabling RNF's used to work